package.json

```diff
@@ -63,9 +64,11 @@
     "@dfinity/utils": "^0.0.20",
     "bip39": "^3.0.4",
     "buffer": "^6.0.3",
+    "dompurify": "^3.0.6",
     "idb-keyval": "^6.2.1",
     "jose": "^5.1.3",
     "lit-html": "^2.7.2",
+    "marked": "^11.0.0",
```

Comment:
out of curiosity, you did not needed `@types/marked`?
I added those in my pet weekly project, so gonna remove it if useless

Reply:
apparently not; I saw a comment about type issues in the html utils from NNS dapp but here it seems like everything worked out of the box

Reply:
Indeed, I have this locally: `node_modules/marked/lib/marked.d.ts`

Reply:
Coolio, thanks